apps/web-giddh/src/app/shared/triggers/components/advance-trigger/advance-trigger.component.ts

```
@@ -0,0 +1,830 @@
+            }
+        });
+    }
+
+    /**
+     * Get campaign fields
+     *
+     * @param {string} slug
+     * @param {Function} [callback]
+     * @memberof AdvanceTriggerComponent
+     */
+    public getCampaignFields(slug: string, callback?: Function): void {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)